assets/js/modules/analytics-4/datastore/properties.js

```diff
@@ -230,22 +230,27 @@ const baseActions = {
 		},
 		function* ( propertyID ) {
 			const registry = yield Data.commonActions.getRegistry();
+			const {
+				setPropertyCreateTime,
+				setSettings,
+				setWebDataStreamID,
+				updateSettingsForMeasurementID,
+			} = registry.dispatch( MODULES_ANALYTICS_4 );
+
+			setSettings( {
+				propertyID,
+				propertyCreateTime: 0,
+			} );
```

Comment:
We should be doing this more actually as we learned in https://github.com/google/site-kit-wp/blob/6a3a0eaa364f7e55352d18398bd2ac1d8cf0dbb7/assets/js/modules/analytics-4/datastore/properties.js#L549-L554

It's not specifically relevant to the fix though.